src/main/java/de/tum/in/www1/artemis/service/StudentExamService.java

```diff
@@ -112,4 +113,29 @@ public StudentExam findOneByExerciseIdAndUserId(Long exerciseId, Long userId) {
         return studentExamRepository.findByExerciseIdAndUserId(exerciseId, userId)
                 .orElseThrow(() -> new EntityNotFoundException("Student exam for exercise " + exerciseId + " and user " + userId + " does not exist"));
     }
+
+    /**
+     * Get the maximal working time of all student exams for the exam with the given id.
+     *
+     * @param examId the id of the exam
+     * @return the maximum of all student exam working times for the given exam
+     * @throws EntityNotFoundException if no student exams could be found
+     */
+    @NotNull
+    public Integer findOneByExerciseIdAndUserId(Long examId) {
```

Comment:
I think the method name is confusing, it should not be `findOne`

Reply:
Yes, this does not fit at all, something went wrong here during editing. I changed it now, thanks!